app/src/main/java/com/nervousfish/nervousfish/activities/BluetoothConnectionActivity.java

```diff
@@ -246,6 +279,7 @@ public void onItemClick(final AdapterView<?> av, final View v, final int arg2, f
             // Get the device MAC address, which is the last 17 chars in the View
             final String info = ((TextView) v).getText().toString();
             if (!info.equals(getString(R.string.no_devices_found))) {
+                isMaster = true;
```

Comment:
Why is `isMaster` being set to true when the string equals to `R.string.no_devices_found`?